nl_server/embeddings_store.py

```diff
@@ -39,7 +39,7 @@ def __init__(self, indexes: List[EmbeddingsIndex]):
 
     # If a new custom index is loaded after startup, it will be merged with the
     # original default index maintained under this variable.
-    self.original_default_idx = dataclasses.replace(default_idx)
+    self.original_default_idx = copy.copy(default_idx)
```

Comment:
To confirm, you don't want deepcopy?

Reply:
It's not a compound object so should not be needed. But changed to `deepcopy` in case it is in the future.